.github/workflows/vib-early-feedback.yaml

```diff
@@ -63,25 +64,28 @@ jobs:
       # Source: https://github.community/t/no-output-on-process-completed-with-exit-code-1/123821/3
       - id: show-error
         name: 'Show error'
-        if: ${{steps.get-chart.outputs.result == 'fail'}}
+        if: ${{ steps.get-chart.outputs.result == 'fail' }}
         uses: actions/github-script@v3
         with:
           script: |
-              core.setFailed('${{steps.get-chart.outputs.error}}')
+              core.setFailed('${{ steps.get-chart.outputs.error }}')
   vib-execution:
     runs-on: ubuntu-latest
     needs: get-chart
-    if: ${{needs.get-chart.outputs.result == 'ok'}}
-    name: 'Package, Lint and Trivy scan'
+    if: ${{ needs.get-chart.outputs.result == 'ok' }}
+    name: 'Package, Lint, Trivy scan & Deployment'
```

Comment:
I've used this name though not every early-feedback pipeline will perform a deployment (there are assets for which we are not performing the `health-check` action). Nevertheless, performing a deployment is the norm.